src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Listeners/BlobTriggerExecutor.Logger.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Blobs.Listeners
+{
+    internal partial class BlobTriggerExecutor
+    {
```

Comment:
Nit: should this be sealed class similar to other partial classes you added for loggers?

Reply:
No, I only did that when the other part of the class was also sealed. Although that wasn't strictly required: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods

> If any part is declared abstract, then the whole type is considered abstract. If any part is declared sealed, then the whole type is considered sealed. If any part declares a base type, then the whole type inherits that class.